web/src/narrow_state.ts

```diff
@@ -398,14 +398,9 @@ export function narrowed_by_stream_reply(current_filter: Filter | undefined = fi
 }
 
 export function is_for_stream_id(stream_id: number, filter?: Filter): boolean {
-    // This is not perfect, since we still track narrows by
-    // name, not id, but at least the interface is good going
-    // forward.
     const narrow_sub = stream_sub(filter);
-
     if (narrow_sub === undefined) {
         return false;
     }
-
     return stream_id === narrow_sub.stream_id;
```

Comment:
This is probably cleaner calling `narrow_state.stream_id`, since we don't actually need the `sub` object, just the ID.

Reply:
Made the update so that this calls `narrow_state.stream_id` as we don't need to worry about whether this is a known channel ID or not for the callers of this function.